### PR TITLE
Fix nuget packages build to include metadata.

### PR DESCRIPTION
**Pull request by michaeljbaird**
Added MetaData to use new csproj format

### CODE REVIEW

**Comment by michaeljbaird** [comment]
@asbjornu It appears the pre-build worked and has [valid nuget metadata now](https://ci.appveyor.com/project/sentry/raven-csharp/build/57.Branch.pull/220/merge.Sha.daa7cd14023c88d6802014a44fea318238a92d8d/artifacts).

File: build.cake [hunk 1]
```diff
@@ -163,8 +163,7 @@ Task("Package")
     {
         EnsureDirectoryExists(artifactsDir);
 
-        foreach (var package in packages)
-        {
+        foreach(var package in packages) {
```
Comment (asbjornu) [inline build.cake]: Can you not change formatting here, please? 😃

File: build.cake [hunk 2]
```diff
@@ -163,7 +163,7 @@ Task("Package")
     {
         EnsureDirectoryExists(artifactsDir);
 
-        foreach (var package in packages)
+        foreach(var package in packages)
```
Comment (asbjornu) [inline build.cake]: Can you please not remove the space between `foreach` and `(` as well? 😄

**Comment by asbjornu** [comment]
Excellent, thanks! Going to attempt pushing out a new build with this in as soon as possible.

**Pull request merged**